build/lib/image_shasum.sh

```diff
@@ -22,10 +22,13 @@ REGISTRY="${1?First argument is registry}"
 REPOSITORY="${2?Second argument is repository}"
 IMAGE_TAG="${3?Third argument is image tag}"
 
-JSONPATH="imageDetails[?(@.imageTags[?(@ == '"${IMAGE_TAG}"')])].imageDigest"
-if [[ "${REGISTRY}" == *"public.ecr.aws"* ]]
+if [ "${REGISTRY}" == "316434458148.dkr.ecr.us-west-2.amazonaws.com" ]
```

Comment:
Should this be changed back?

Reply:
No, we are using skopeo now to get the shasum and this is for in the case of presubmits, we are not going to have a image to get a shasum, we'll just use "latest" so we can test building the helm chart

Reply:
O so it's ok to include the account ID here then?